test/topology_experimental_raft/test_tablets_removenode.py

```diff
@@ -0,0 +1,195 @@
+
+    # RF=3
+    await cql.run_async("CREATE KEYSPACE test3 WITH replication = {'class': 'NetworkTopologyStrategy', 'replication_factor': 3}"
+                        " AND tablets = {'initial': 32};")
+    await cql.run_async("CREATE TABLE test3.test (pk int PRIMARY KEY, c int);")
+
+    logger.info("Populating table")
+
+    keys = range(256)
+    await asyncio.gather(*[cql.run_async(f"INSERT INTO test.test (pk, c) VALUES ({k}, {k});") for k in keys])
+    await asyncio.gather(*[cql.run_async(f"INSERT INTO test2.test (pk, c) VALUES ({k}, {k});") for k in keys])
+    await asyncio.gather(*[cql.run_async(f"INSERT INTO test3.test (pk, c) VALUES ({k}, {k});") for k in keys])
+
+    async def check():
+        # RF=1 table "test" will experience data loss so don't check it.
+        # We include it to check that the system doesn't crash.
+
+        logger.info("Checking table test2")
+        query = SimpleStatement("SELECT * FROM test2.test;", consistency_level=ConsistencyLevel.ONE)
+        rows = await cql.run_async(query)
+        assert len(rows) == len(keys)
+        for r in rows:
+            assert r.c == r.pk
+
+        logger.info("Checking table test3")
+        query = SimpleStatement("SELECT * FROM test3.test;", consistency_level=ConsistencyLevel.ONE)
+        rows = await cql.run_async(query)
+        assert len(rows) == len(keys)
+        for r in rows:
+            assert r.c == r.pk
+
+    await check()
+
+    # Disable migrations concurrent with replace since we don't handle nodes going down during migration yet.
+    # See https://github.com/scylladb/scylladb/issues/16527
+    await manager.api.disable_tablet_balancing(servers[0].ip_addr)
+
+    logger.info('Replacing a node')
+    await manager.server_stop(servers[0].server_id)
+    replace_cfg = ReplaceConfig(replaced_id = servers[0].server_id, reuse_ip_addr = False, use_host_id = True)
+    await manager.server_add(replace_cfg)
+    servers = servers[1:]
+
+    await check()
+
+
+@pytest.mark.asyncio
+async def test_removenode(manager: ManagerClient):
+    logger.info("Bootstrapping cluster")
+    cmdline = ['--logger-log-level', 'storage_service=trace']
+
+    # 4 nodes so that we can find new tablet replica for the RF=3 table on removenode
+    servers = await manager.servers_add(4, cmdline=cmdline)
+
+    cql = manager.get_cql()
+
+    # RF=1
+    await cql.run_async("CREATE KEYSPACE test WITH replication = {'class': 'NetworkTopologyStrategy', 'replication_factor': 1}"
+                        " AND tablets = {'initial': 32};")
+    await cql.run_async("CREATE TABLE test.test (pk int PRIMARY KEY, c int);")
+
+    # RF=2
+    await cql.run_async("CREATE KEYSPACE test2 WITH replication = {'class': 'NetworkTopologyStrategy', 'replication_factor': 2}"
+                        " AND tablets = {'initial': 32};")
+    await cql.run_async("CREATE TABLE test2.test (pk int PRIMARY KEY, c int);")
+
+    # RF=3
```

Comment:
Can we have a helper? This block  duplicated 6 times.

Reply:
We can.